### ISSUE TITLE
NuGet packages for Network projects.

### ISSUE THREAD

**Issue by azchohfi** [mono/MonoGame#3619]
I tried using the NuGet packages and I can't seem to find the "*.Net.*" dll.
I think they should be included, because GamerServices and Guide are inside this assemblies.
At least there should be a second separate NuGet package for the Network assemblies.

**Comment by danzel** [comment]
@ddreaper I think you handle the NuGet packages right?

**Comment by DDReaper** [comment]
Yeah I do, we just didn't get the .Net package done in Tim for the 3.3 release. Will get to them soon

**Comment by SimonDarksideJ** [comment]
Networking was removed from the MonoGame stack due to other dependencies being removed.
However, we are looking to start a Bounty soon to provide a sample to deliver networking, but sadly the .NET package is not likely to come back
